include/aspect/stokes_matrix_free.h

```diff
@@ -57,6 +57,73 @@ namespace aspect
    */
   namespace MatrixFreeStokesOperators
   {
+
+    /**
+     * This struct stores the data for the current linear operator that is requried to perform
+     * matrix-vector products.
+     */
+    template <int dim, typename number>
+    struct OperatorCellData
+    {
+      /**
+       * Information on the compressibility of the flow.
+       */
+      bool is_compressible;
+
+      /**
+       * Pressure scaling constant.
+       */
+      double pressure_scaling;
+
+      /**
+       * If true, Newton terms are part of the operator.
+       */
+      bool enable_newton_derivatives;
+
+      /**
+       * Symmetrize the Newton system when it's true (i.e., the stabilization is symmetric or SPD).
+       */
+      bool symmetrize_newton_system;
+
+      /**
+       * Table which stores viscosity values for each cell.
+       */
+      Table<2, VectorizedArray<number>> viscosity;
```

Comment:
There are a number of 2d tables in this structure. Would you mind documenting in the class doc what the indices of these tables correspond to?